interactive/templatetags/querystring_tools.py

```diff
@@ -0,0 +1,64 @@
+from django import template
+from django.urls import reverse
+from furl import furl
+
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=True)
+def redirect_with_querystring(context, view_name):
```

Comment:
These querystring_tools aren't being used at all in the templates right now. Not sure if they'll be needed?

Reply:
Now in use with the addition of filters to UserList!